src/tests/functional/plugin/cpu/subgraph_tests/src/convs_and_sums.cpp

```diff
@@ -0,0 +1,65 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "ngraph_functions/builders.hpp"
+#include "ngraph_functions/utils/ngraph_helpers.hpp"
+#include "test_utils/cpu_test_utils.hpp"
+
+using namespace ngraph;
+using ngraph::helpers::EltwiseTypes;
+
+namespace SubgraphTestsDefinitions {
+
+class ConvsAndSums : virtual public LayerTestsUtils::LayerTestsCommon {
```

Comment:
Could you please add description for this test. Like draw the graph and specify the test goal in the comments.